src/store/env-config.unit.ts

```diff
@@ -0,0 +1,201 @@
+import { EnvConfig } from "./env-config";
+import { initializeAxios } from "../utils/api";
+import { NuxtAxiosInstance } from "@nuxtjs/axios";
+
+const mockEnvs = {
```

Comment:
is this available as an defined interface?

Reply:
yes here the type is defined https://github.com/hpi-schul-cloud/nuxt-client/blob/9d15977dccd7f2b339d3592aec0164ce68b67b7c/src/store/env-config.ts#L20

Reply:
then i would suggest using it like
```suggestion
const mockEnvs: Envs = {
```